### PR TITLE
SONAR-6736 refactor step names and descriptions

### PR DESCRIPTION
**Pull request by simonbrandhof**
- use imperative verbs instead of nouns when possible
- improve descriptions: more simple and more consistent with class name
- fix some copy-pasted descriptions

### CODE REVIEW

File: server/sonar-server/src/test/java/org/sonar/server/computation/step/LoadQualityProfilesStepTest.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 import static org.sonar.db.rule.RuleTesting.XOO_X1;
 import static org.sonar.db.rule.RuleTesting.XOO_X2;
 
-public class FeedActiveRulesStepTest {
+public class LoadQualityProfilesStepTest {
 
   @Rule
   public TemporaryFolder temp = new TemporaryFolder();
```
Comment (SonarTech) [inline server/sonar-server/src/test/java/org/sonar/server/computation/step/LoadQualityProfilesStepTest.java]: MAJOR Remove this unused "TemporaryFolder". [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2924)

**Comment by SonarTech** [comment]
SonarQube analysis reported 13 issues:
* CRITICAL 1 critical
* MAJOR 2 major
* MINOR 10 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/CommentMeasuresStep.java#L114) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/CommentMeasuresStep.java#L113) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/CommentMeasuresStep.java#L155) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/CommentMeasuresStep.java#L157) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeMeasureVariationsStep.java#L167) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeMeasureVariationsStep.java#L167) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MAJOR [The Cyclomatic Complexity of this method "computeVariation" is 11 which is greater than 10 authorized.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeMeasureVariationsStep.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/DuplicationMeasuresStep.java#L116) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/DuplicationMeasuresStep.java#L116) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* CRITICAL [Either log or rethrow this exception.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/LoadPeriodsStep.java#L267) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/UnitTestMeasuresStep.java#L115) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/560fd668131d2635e94c514758cd40952ad16515/server/sonar-server/src/main/java/org/sonar/server/computation/step/UnitTestMeasuresStep.java#L116) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)

**Comment by sns-seb** [comment]
@SonarTech has some good points

**Comment by simonbrandhof** [comment]
@sns-seb of course! What did you expect ? :-)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 1 major
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/166cedb128da8ea4a9b99d6ed78e32fc3829a996/server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeMeasureVariationsStep.java#L167) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)
* MAJOR [The Cyclomatic Complexity of this method "computeVariation" is 11 which is greater than 10 authorized.](https://github.com/SonarSource/sonarqube/blob/166cedb128da8ea4a9b99d6ed78e32fc3829a996/server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeMeasureVariationsStep.java#L158) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
* MINOR [Upper-case this literal "d" suffix.](https://github.com/SonarSource/sonarqube/blob/166cedb128da8ea4a9b99d6ed78e32fc3829a996/server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeMeasureVariationsStep.java#L167) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS818)

**Pull request closed without merging**